services/order.py

```diff
@@ -0,0 +1,32 @@
+from django.db.models import QuerySet
+from datetime import datetime
+from django.db import transaction
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(tickets: list[dict], username: str, date: str = None) -> None:
+    with transaction.atomic():
+        user = User.objects.get(username=username)
```

Comment:
Consider handling exceptions such as `User.DoesNotExist` when retrieving the `User` instance. This will prevent the application from crashing if an invalid `username` is provided.